components/Alert/index.jsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+
+export const alertTestIds = {
+  icon: 'alert-icon',
+  title: 'alert-title',
+  description: 'alert-description',
+};
+
+const Alert = ({ icon, title, description }) => {
+  return (
+    <div className="w-full bg-notification backdrop-blur-2.5xl flex p-4 rounded-xl">
```

Comment:
How about adding a test id for this outer div so when working on the backend part we can use it to check whether the alert is visible or not? 💡 
And maybe rename other test ID also 💭 

```javascript
export const alertTestIds = {
  base: 'alert',
  icon: 'alert__icon',
  title: 'alert__title',
  description: 'alert__description',
};
```

Reply:
done in 98bcf8c4034b7b2509d56f4680084e3416499844